build/storage/tests/it/test-drivers/init.hot-plug.nvme

```diff
@@ -60,7 +65,9 @@ verify_expected_number_of_nvme_namespaces "$vm_serial" 2
 
 echo ""
 echo "### Delete nvme device ###"
-delete_nvme_device "$ipu_storage_container_ip" "$nvme1"
+delete_nvme_device "$ipu_storage_container_ip" \
+    "$DEFAULT_SMA_PORT" "$host_target_ip" "$DEFAULT_HOST_TARGET_SERVICE_PORT" \
+	"$nvme1"
```

Comment:
I assume won't run into a problem here in a solution under our control but in theory there could be other nvme0 or nvme1 devices and our numbering would be starting at the first existing NVMe device, right?

Reply:
nvme1 here is not a name of a device on host
It is a variable with SMA handle and when we delete a device, it can point to for example /dev/nvme35
So I think we are safe here